Tests/scripts/pkg_dev_tasks_in_docker.py

```diff
@@ -115,11 +115,9 @@ def docker_run(project_dir, docker_image, no_test, no_lint, keep_container):
     pylint_files = get_pylint_files(project_dir)
     # copy demistomock and common server
     shutil.copy(CONTENT_DIR + '/Tests/demistomock/demistomock.py', project_dir)
-    open(project_dir + '/CommonServerUserPython.py', 'a').close()  # create empty file
     shutil.rmtree(project_dir + '/__pycache__', ignore_errors=True)
-    subprocess.check_call(['python2', CONTENT_DIR + '/package_extractor.py', '-i',
-                           'Scripts/script-CommonServerPython.yml', '-o',
-                           project_dir + '/CommonServerPython.py'], cwd=CONTENT_DIR)
+    subprocess.check_call(['cp', CONTENT_DIR + '/Scripts/CommonServerPython/CommonServerPython.py',
+                           CONTENT_DIR + '/CommonServerPython.py'], cwd=CONTENT_DIR)
```

Comment:
We need the file in the project dir for the way we run within docker.
```suggestion
                           project_dir + '/CommonServerPython.py'], cwd=CONTENT_DIR)
```